tests/kafkatest/tests/produce_consume_validate.py

```diff
@@ -103,13 +103,14 @@ def stop_producer_and_consumer(self):
         self.producer.stop()
         self.consumer.wait()
 
-    def run_produce_consume_validate(self, core_test_action=None, *args):
-        """Top-level template for simple produce/consume/validate tests."""
+    def run_multistep_produce_consume_validate(self, core_test_actions, *args):
```

Comment:
Are these changes still needed now that the downgrade test is extending EndToEndTest?

Reply:
Thanks, I'll revert these changes.